src/commands/Minion/nightmare.ts

```diff
@@ -184,6 +255,21 @@ export default class extends BotCommand {
 			} else {
 				effectiveTime *= 0.96;
 			}
+			if (isPhosani) {
+				effectiveTime = reduceNumByPercent(effectiveTime, 25);
```

Comment:
Just a suggestion :)
I think this should be maybe 30-33%, a relatively slow phosani killl is 12 minutes. In OSRS I got a 9:10 on my 19th phosani kill. So 3 kills in 40 minutes with tablet seems a little slow. I also had none of the boosts below in game, just using godsword and bludgeon.